Fix hardcoded paths to perl.
We depend on EasyBuild Perl.

Åke Sandgren, 20210824
diff -ru elpa-new_release_2021.05.002_bugfix.orig/fdep/fortran_dependencies.pl elpa-new_release_2021.05.002_bugfix/fdep/fortran_dependencies.pl
--- elpa-new_release_2021.05.002_bugfix.orig/fdep/fortran_dependencies.pl	2021-06-07 11:35:08.000000000 +0200
+++ elpa-new_release_2021.05.002_bugfix/fdep/fortran_dependencies.pl	2021-08-24 18:15:00.133701035 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 #
 # Copyright 2015 Lorenz Hüdepohl
 #
diff -ru elpa-new_release_2021.05.002_bugfix.orig/test_project_1stage/fdep/fortran_dependencies.pl elpa-new_release_2021.05.002_bugfix/test_project_1stage/fdep/fortran_dependencies.pl
--- elpa-new_release_2021.05.002_bugfix.orig/test_project_1stage/fdep/fortran_dependencies.pl	2021-06-07 11:35:08.000000000 +0200
+++ elpa-new_release_2021.05.002_bugfix/test_project_1stage/fdep/fortran_dependencies.pl	2021-08-24 18:15:33.421492236 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 use strict;
 
diff -ru elpa-new_release_2021.05.002_bugfix.orig/test_project_2stage/fdep/fortran_dependencies.pl elpa-new_release_2021.05.002_bugfix/test_project_2stage/fdep/fortran_dependencies.pl
--- elpa-new_release_2021.05.002_bugfix.orig/test_project_2stage/fdep/fortran_dependencies.pl	2021-06-07 11:35:08.000000000 +0200
+++ elpa-new_release_2021.05.002_bugfix/test_project_2stage/fdep/fortran_dependencies.pl	2021-08-24 18:15:36.101475423 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 use strict;
 
diff -ru elpa-new_release_2021.05.002_bugfix.orig/test_project_C/fdep/fortran_dependencies.pl elpa-new_release_2021.05.002_bugfix/test_project_C/fdep/fortran_dependencies.pl
--- elpa-new_release_2021.05.002_bugfix.orig/test_project_C/fdep/fortran_dependencies.pl	2021-06-07 11:35:08.000000000 +0200
+++ elpa-new_release_2021.05.002_bugfix/test_project_C/fdep/fortran_dependencies.pl	2021-08-24 18:15:38.653459416 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 use strict;
